manim/mobject/graph.py

```diff
@@ -679,8 +679,11 @@ def _populate_edge_dict(
         """Helper method for populating the edges of the graph."""
         raise NotImplementedError("To be implemented in concrete subclasses")
 
-    def __getitem__(self: Graph, v: Hashable) -> Mobject:
-        return self.vertices[v]
+    def __getitem__(self: Graph, k: Hashable | tuple[Hashable, Hashable]) -> Mobject:
+        if isinstance(k, tuple):
+            return self.edges[k]
+        else:
+            return self.vertices[k]
```

Comment:
I think a better error message might be useful, so what about something like
```suggestion
        if isinstance(k, tuple):
            if (edge := self.edges.get(k)) is not None:
                return edge
        elif (vertex := self.vertices.get(k)) is not None:
            return vertex
        raise ValueError(f"Could not find {k} in vertices or edges")
```